cli/src/main/resources/config.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We are not using the Spring Boot any more in cli, thus we are using injection from xml.